docs/documentation/helpcenter.md

```diff
@@ -148,6 +152,32 @@ with modals, etc. should never go in the help center documentation.
 In such cases, you may be able to fix the problem by adding text in-app,
 where the user will see it as they are interacting with the feature.
 
+### Redirecting an existing article
+
+From time to time, there is a reason to rename articles in the help center or
+REST API documentation. This breaks incoming links, including from other
+branches of the repository that haven't been rebased, and more importantly
+from previous versions of Zulip.
```

Comment:
Thanks for these suggestions. I think Tim meant that the blog would be a source for broken links and not something that we would rename/redirect. Also, since there are multiple repositories (zulip-mobile, zulip-desktop, etc.), I thought your rewording was a bit confusing. The updated version is now ...

> From time to time, we might want to rename an article in the help
center, or REST API documentation. This change will break incoming
links, including links in published Zulip blog posts, links in other
branches of the repository that haven't been rebased, and more
importantly links from previous versions of Zulip.